recipes/ldraw/build_mklist.sh

```diff
@@ -0,0 +1,21 @@
+set -e
+unzip -d mklist -o mklist1_6.zip
+
+if [ $(uname) == Darwin ]; then
+    INPLACE_SED="sed -i \"\" -e"
```

Comment:
You can add `sed` to the `build` requirements to get GNU sed on macOS. It's also preferable on Linux instead of assuming `sed` is installed on on the host.